### ISSUE TITLE
[Harbor] Set postgres password via secret in harbor

### ISSUE THREAD

**Issue by lusu007** [bitnami/charts#5468]
Unfortunately I have not found a suitable issue template for my question.

I would like to change the default password for the postgres instance that is installed with harbor. Is this only possible directly via values.yaml or also via secret?

**Comment by javsalgar** [comment]
Hi,

You could use the `postgresql.postgresqlPassword` section from the postgresql subchart. Check the bitnami/postgresql documentation for the full list of values https://github.com/bitnami/charts/tree/master/bitnami/postgresql

Remember to set the `postgresql.` prefix as it is a subchart in Harbor.

**Comment by lusu007** [comment]
I have also seen this variable. My problem is that I am looking for a way to set the password by creating secret. For me, all settings are stored in git and I would not like to have passwords in there, even if it is only the default password.

Normally I would create a sealedsecret and add it as an existingSecret to my helm chart.

**Comment by miguelaeh** [comment]
Hi @lusu007 ,
It seems that it is not possible right now to set the Postgresql password using an existing secret. Would you like to create a PR?

**Comment by lusu007** [comment]
I'll try it in the days, but without guarantee that I get it done 😅😊

**Comment by lusu007** [comment]
I just tried something but I dont know how to test my changes. Do you an example how to test probably?

**Comment by miguelaeh** [comment]
To test the changes you will need to create the secret, and then install the local chart using something like the followiing:
```
helm install <name> . --set <theNewParam>=<secretName>
```

That command should be executed from the chart directory that's why I used `.`, you can also execute it from other directory changing the `.` by the path.

Then connect to PostgreSQL and check that the password is the one inside the secret you previously created

**Comment by miguelaeh** [comment]
Hi @lusu007 ,
Thank you very much for the collaboration.
To test the changes you will need to create the secret, and then install the local chart using something like the followiing:
```
helm install <name> . --set <theNewParam>=<secretName>
```

That command should be executed from the chart directory that's why I used `.`, you can also execute it from other directory changing the `.` by the path.

Then connect to PostgreSQL and check that the password is the one inside the secret you previously created

**Comment by lusu007** [comment]
That's what I did. But I got an error that my Chart dependencies were not found.
If you need I can provide more information in a few days.

**Comment by miguelaeh** [comment]
aah, you need to execute `helm dependencies update` from the chart directory before be able to install it to fetch the subcharts.

**Comment by lusu007** [comment]
Ah okay thanks! I will try it tomorrow.

**Comment by lusu007** [comment]
Merged in #5542 .
